docs/examples/DataSet/Working with snapshots.ipynb

```diff
@@ -620,9 +619,9 @@
  ],
  "metadata": {
   "kernelspec": {
-   "display_name": "Python [conda env:qcodes]",
+   "display_name": "Python 3",
    "language": "python",
-   "name": "conda-env-qcodes-py"
+   "name": "python3"
```

Comment:
i am no sure where to look for it, but is the kernelspec now still ignored or not? when the notebooks are executed, what kernel is used?

Reply:
perfect! let's use the solution.

for `jupyter-nbconvert` we used to pass `--ExecutePreprocessor.kernel_name=python3` so that the kernel mentioned in the notebooks is ignored.

note that we will never be able to get rid of the various kernelspecs in the notebooks, because when we edit them, we run them on our own PCs and setups, hence we select a particular kernel for that, and if a developer explicitly does not change it back to `python 3` or whatever, his kernelspec gets saved.